FileIO/GmshIO/GMSHPolygonTree.cpp

```diff
@@ -86,7 +86,11 @@ void GMSHPolygonTree::insertPolyline (GeoLib::PolylineWithSegmentMarker * ply)
 			if (! ply->isSegmentMarked(k)) {
 				std::size_t seg_num(0);
 				GeoLib::Point *intersection_pnt(new GeoLib::Point);
```

Comment:
Good point. Will include your remark in this [GMSHInterface-branch](https://github.com/TomFischer/ogs/tree/GMSHInterface).